app/controllers/serviceLearning/routes.py

```diff
@@ -112,28 +116,42 @@ def withdrawCourse(courseID):
         else:
             flash("Unauthorized to perform this action", 'warning')
     except Exception as e:
+        print(e)
         flash("Withdrawal Unsuccessful", 'warning')
     return ""
 
+@serviceLearning_bp.route('/serviceLearning/renew/<courseID>/<termID>/', methods = ['POST'])
+def renewCourse(courseID, termID):
+    try:
+        if g.current_user.isCeltsAdmin:
```

Comment:
Faculty need to be able to renew their own courses

Reply:
resolved